src/kata3.numberToReversedDigits.js

```diff
@@ -1,3 +1,6 @@
-const numberToReversedDigits = number => {};
+const numberToReversedDigits = number => {
+  const myArray = Array.from(number.toString()).map(Number);
+  return myArray.reverse();
```

Comment:
Nice, you can apply your reverse directly in a chain after your map too if you didn't want to assign your array to your myArray variable and then return that.